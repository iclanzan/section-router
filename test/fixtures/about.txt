# About

The **txt** extension should also be picked up.
