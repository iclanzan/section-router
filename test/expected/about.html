<p>The <strong>txt</strong> extension should also be picked up.</p>
